README.md

```diff
@@ -47,3 +48,5 @@ Add yours via a pull request!
 - [Waku](https://waku.gg/)
 - [Ember](https://emberjs.com/)
 - [Playwright](https://playwright.dev/)
+- [Stormkit Monorepo React](https://github.com/stormkit-io/monorepo-template-react)
```

Comment:
```suggestion
```

I'll remove this for now as I don't think we'd want to keep templates in the list at the moment.